WBooks/Views/BookIndex/BookInformation/BookInformationViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  BookInformationViewController.swift
+//  WBooks
+//
+//  Created by Diego Quirós on 3/19/18.
+//  Copyright © 2018 Wolox. All rights reserved.
+//
+
+import UIKit
+import Core
+import ReactiveSwift
+import enum Result.NoError
+
+class BookInformationViewController: UIViewController {
+    
+    private lazy var _view: BookInformationView = BookInformationView.loadFromNib()!
+    
+    var bookViewModel: BookViewModel
+    var bookInformationViewModel: BookInformationViewModel
+
+    init(bookViewModel viewModel: BookViewModel) {
+        self.bookViewModel = viewModel
+        self.bookInformationViewModel = BookInformationViewModel(bookViewModel: viewModel)
+        super.init(nibName: nil, bundle: nil)
+        bindViewModel()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        
+        navigationItem.title = "BOOK DETAIL"
+        super.viewDidLoad()
+        configureTableView()
+        _view.configureBookInformation()
+        
+    }
+
+    override func loadView() {
+        view = _view
+    }
+
+}
+
+extension BookInformationViewController: UITableViewDelegate, UITableViewDataSource {
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        if bookInformationViewModel.count > 5 {
```

Comment:
Why did you do this?

Reply:
https://trello.com/c/FqVkQGgS/20-tabla-para-comentarios
this card tells you to show the last 5 comments

Reply:
I would recommend to do that inside the view model. Now that I double check the view model you are only taking 1 comment Why?

https://github.com/wolox-training/dq-swift/blob/39bfa791706c4527c6eb1f3f4fa8031424cc0bf3/WBooks/Views/BookIndex/BookInformation/BookInformationViewModel.swift#L33

Reply:
that's a mistake, i'll change it